### ISSUE TITLE
ADA Search Box: Placeholder text lacks contrast ratio (Major / Easy)

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#890]
Part of #324

Deque Issues: 136197, 137164, 137754

## Problem
- Placeholder text lacks 4.5:1 contrast ratio

## Solution
- Darken placeholder text
